.pfnci/install_cupy.bat

```diff
@@ -0,0 +1,23 @@
+@echo off
+
+set CHAINER_LATEST_MAJOR_VER=7
```

Comment:
Should we update the version number on each major release?
How about other approach?
(What I come up with immediately is to check presence of the branch `v%CHAINER_MAJOR_VER%` after clone).